.ci/run.sh

```diff
@@ -5,7 +5,7 @@ set -e
 # move to Kibana root
 cd "$(dirname "$0")/.."
 
-source src/dev/ci_setup/extract_bootstrap_cache.sh
+#source src/dev/ci_setup/extract_bootstrap_cache.sh
```

Comment:
Are you intending to leave this off for any reason?

Reply:
No, I need to uncomment that in conjunction with a once-off operation that @spalger is going to take care of tomorrow at merge-time. It was necessary right now in order to get tests to run in these branches because the packer cache is using the old plugin paths, which is still appropriate for all other PRs until this merges.

So basically it's a chicken or the egg situation in both of these PRs.

Reply:
I split out the proper temporary change into it's own commit: d04d8a285661af64ce52bdb9f95c234ed0574321